app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -41,17 +41,17 @@ object FeatureFlags {
     /**
      * Enables the "recent" tabs feature in the home screen.
      */
-    val showRecentTabsFeature = Config.channel.isNightlyOrDebug
+    val showRecentTabsFeature = Config.channel.isNightlyOrDebug || Config.channel.isBeta
 
     /**
      * Enables UI features based on history metadata.
      */
-    val historyMetadataUIFeature = Config.channel.isNightlyOrDebug
+    val historyMetadataUIFeature = Config.channel.isNightlyOrDebug || Config.channel.isBeta
 
     /**
      * Enables the recently saved bookmarks feature in the home screen.
      */
-    val recentBookmarksFeature = Config.channel.isNightlyOrDebug
+    val recentBookmarksFeature = Config.channel.isNightlyOrDebug || Config.channel.isBeta
```

Comment:
I didn't active pocket in Beta as it was enabled yesterday on nightly for some specific users, see `isPocketRecommendationsFeatureEnabled`, we have to chat we product to see if they want to enable it right away.